websites.yml

```diff
@@ -144,3 +144,6 @@ avatar:
   tumblr: opengraph
   twitter:
     url: https://twitter.com/{u}/profile_image?size=original
+icon_css:
```

Comment:
this `icon_css:` structure / design is not declarative enough.  It describes the implementation.

The purpose of this file is to remove any 'meaning' and just include re-usable facts.

The 'bug' at the moment is that the website key used throughout this file is also the fontawesome name.

Which isnt good enough, all the time.

So we need a new section of the yaml which provides facts about logos for the website.

```yaml
logos:
  wikimedia:
    fontawesome: wikipedia-fa
  keybase:
    url: https://png.icons8.com/ios/30/000000/keybase.png
```

Reply:
I think the title should be "declare logo information" rather than "adding functionality to customize".